tests/openwisp2/settings.py

```diff
@@ -215,6 +222,12 @@
     GEO_DEVICELOCATION_MODEL = 'sample_geo.DeviceLocation'
     CONNECTION_CREDENTIALS_MODEL = 'sample_connection.Credentials'
     CONNECTION_DEVICECONNECTION_MODEL = 'sample_connection.DeviceConnection'
+    SUBNET_DIVISION_SUBNETDIVISIONRULE_MODEL = (
+        'sample_subnet_division.SubnetDivisionRule'
+    )
+    SUBNET_DIVISION_SUBNETDIVISIONINDEX_MODEL = (
+        'sample_subnet_division.SubnetDivisionIndex'
+    )
```

Comment:
The readme section which explains how to extend ow-controller needs to be updated as well.